### PR TITLE
Bump core, memory limits

### PR DESCRIPTION
**Pull request by TomAugspurger**
xref https://github.com/pangeo-data/pangeo-cloud-federation/issues/619#issuecomment-642278121

### CODE REVIEW

**Comment by jhamman** [comment]
@TomAugspurger - FWI, I think we should just do this in the pangeo-cloud-federation sub-charts. Presumably, every hub there will want to configure this differently.

File: pangeo/values.yaml [hunk 1]
```diff
@@ -100,8 +100,8 @@ dask-gateway:
                 "image": options.image,
             }
         c.Backend.cluster_options = Options(
-            Integer("worker_cores", 2, min=1, max=4, label="Worker Cores"),
-            Float("worker_memory", 4, min=1, max=8, label="Worker Memory (GiB)"),
+            Integer("worker_cores", 2, min=1, max=16, label="Worker Cores"),
+            Float("worker_memory", 4, min=1, max=32, label="Worker Memory (GiB)"),
```
Comment (rabernat) [inline pangeo/values.yaml:104]: `max=32` 😁

**Comment by chiaral** [comment]
Hello there - what is the timeline to merge this? Even just in staging? thanks!

**Comment by TomAugspurger** [comment]
Let me apply these changes just to pangeo-cloud-federation.

**Pull request closed without merging**